src/bin/gui/connected_ast.ml

```diff
@@ -772,20 +771,20 @@ and popup_trigger t qid env (sbuf:sbuffer) offset () =
   ignore(GMisc.image ~stock:`CANCEL ~packing:phbox#add ());
   ignore(GMisc.label ~text:"Cancel" ~packing:phbox#add ());
 
-  let lmanager = GSourceView2.source_language_manager ~default:true in
+  let lmanager = GSourceView3.source_language_manager ~default:true in
   let source_language = lmanager#language "alt-ergo" in
   let buf1 = match source_language with
     | Some language ->
-      GSourceView2.source_buffer ~language
+      GSourceView3.source_buffer ~language
         ~highlight_syntax:true ~highlight_matching_brackets:true ()
-    | None -> GSourceView2.source_buffer () in
+    | None -> GSourceView3.source_buffer () in
   let sw1 = GBin.scrolled_window
       ~vpolicy:`AUTOMATIC
       ~hpolicy:`AUTOMATIC
       ~packing:pop_w#vbox#add () in
-  let tv1 = GSourceView2.source_view ~source_buffer:buf1 ~packing:(sw1#add)
-      ~show_line_numbers:true ~wrap_mode:`CHAR() in
-  let _ = tv1#misc#modify_font monospace_font in
+  let tv1 = GSourceView3.source_view ~source_buffer:buf1 ~packing:(sw1#add)
+              ~show_line_numbers:true ~wrap_mode:`CHAR() in
```

Comment:
Good coding habit: `in` should close the code block. You either have `let` & `in` on the same line or starting on the same column.
Ex: 
```ocaml
let x = 
  let y = f z in
  y + 1
in ...
```